featurebyte/__init__.py

```diff
@@ -165,6 +166,60 @@ def use_profile(profile: str) -> None:
             pass
 
 
+def register_tutorial_api_token(api_token: str) -> None:
+    """
+    Register tutorial api token, and use the tutorial profile. This will write a new tutorial profile to the
+    configuration file.
+
+    Parameters
+    ----------
+    api_token: str
+        Tutorial api token
+
+    Examples
+    --------
+    >>> fb.register_tutorial_api_token("your_api_token")  # doctest: +SKIP
+    """
+    # Read configuration file
+    config_file_path = Configurations().config_file_path
+    with open(config_file_path, encoding="utf-8") as file_obj:
+        loaded_config = yaml.safe_load(file_obj)
+
+    profiles: List[Any] = loaded_config["profile"]
+    tutorial_profile_name = "tutorial"
+    has_tutorial_profile = False
+    updated_profile = False
+    # Update API token in existing profile if it's there
+    for profile in profiles:
+        if profile["name"] == tutorial_profile_name:
+            if "api_token" in profile:
+                updated_profile = profile["api_token"] != api_token
+            else:
+                updated_profile = True
+            profile["api_token"] = api_token
+            has_tutorial_profile = True
```

Comment:
nit: can consider breaking from this loop once tutorial profile name is found and replace `if not has_tutorial_profile:` with `else:` then there's no need for the variable `has_tutorial_profile`.

Reply:
😱 TIL. for-loop-break-else